implementations/rust/ockam/ockam_identity/src/identities/storage/attributes_entry.rs

```diff
@@ -56,3 +58,21 @@ impl AttributesEntry {
         self.attested_by.to_owned()
     }
 }
+
+impl AttributesEntry {
+    /// Create an AttributesEntry with just one name/value pair
+    pub fn singleton(
```

Comment:
The name is a little bit misleading

Reply:
This is a common name in Haskell-land for this sort of thing. What do you suggest then?